tendrl/common/manager/rpc_job_process.py

```diff
@@ -57,6 +57,12 @@ def _acceptor(self, job_type):
                     continue
                 raw_job = json.loads(job.value.decode('utf-8'))
                 try:
+                    # If current node's id does not fall under job's node_ids,
+                    # ignore the job and dont process
+                    if "node_ids" in raw_job:
```

Comment:
this would mean there be "node_ids" in even jobs intended for the integration processes (gluster, ceph), why do you want that?